src/TLuaInterpreter.cpp

```diff
@@ -13769,3 +13770,40 @@ int TLuaInterpreter::startPermSubstringTrigger(const QString & name, const QStri
 
 }
 
+int TLuaInterpreter::alert( lua_State * L )
+{
+    int luaAlertDuration;
+
+    int n = lua_gettop( L );
+    if( n == 0 )
+    {
+        // msec argument for QApplication::alert() is optional (defaults to 0)
+        luaAlertDuration = 0;
+    }
+    else
+    {
+        if( ! lua_isnumber( L, 1 ) )
+        {
+            lua_pushfstring( L, "alert: bad argument #1 type (alert duration in seconds as number expected, got %s!)",
+                            luaL_typename( L, 1 ));
+            lua_error( L );
+            return 1;
```

Comment:
... one of the external code review sites that is linked into the Mudlet codebase now can get a bit funny as `lua_error(L);` never actually returns so it moans about unreachable code.  It is nicely silenced by a new code blob that you may see replacing the above two lines as:

    return lua_error(L);

...